testing/btest/scripts/base/protocols/dns/tkey.zeek

```diff
@@ -0,0 +1,10 @@
+# @TEST-EXEC: zeek -b -C -r $TRACES/dns/tkey.pcap %INPUT > output
+# @TEST-EXEC: btest-diff dns.log
+# @TEST-EXEC: btest-diff output
+
+@load policy/protocols/dns/auth-addl
```

Comment:
Any reason this policy script is loaded?

```suggestion
@load base/protocols/dns
```

Reply:
So with the policy script there are two `TKEY` events triggered in the baseline (with field-by-field output):

```
TKEY
query, 1068-ms-7.309-2c6e448.7a9463b8-b109-11ed-26a3-080027f220e5
qtype, 3
alg_name, gss-tsig
inception, 1676938156.0
expiration, 1677024556.0
mode, 3
rr_error, 0
key_data size, 3073
is_query, 1
TKEY
query, 1068-ms-7.309-2c6e448.7a9463b8-b109-11ed-26a3-080027f220e5
qtype, 1
alg_name, gss-tsig
inception, 1676938118.0
expiration, 1677024518.0
mode, 3
rr_error, 0
key_data size, 185
is_query, 0
```

With `base/protocols/dns` there's only one:

```
TKEY
query, 1068-ms-7.309-2c6e448.7a9463b8-b109-11ed-26a3-080027f220e5
qtype, 1
alg_name, gss-tsig
inception, 1676938118.0
expiration, 1677024518.0
mode, 3
rr_error, 0
key_data size, 185
is_query, 0
```

Since I couldn't confirm why the first one was dropped I went with the one that gave more to test, but happy to change that if there's a particular reason I should know.

Reply:
Ah! Seems the `dns_skip_all_addl` defaults to `T`. Can check the `tsig.zeek` test setting it to `F`:
```
redef dns_skip_all_addl = F;
```

Hmm, I wonder a bit if for TKEY this flag should be ignored (the addl rr seems a crucial part), but that's for another time.